docs/specs/target.md

```diff
@@ -0,0 +1,188 @@
+  [aggregator architecture requirements](#aggregator-architecture).
+- Targets MAY support the [unified architecture][unified_architecture] by
+  providing a single command that deploys Vector and achieves the
+  [unified architecture requirements](#unified-architecture).
+
+### 4. Agent Architecture
+
+The [agent architecture][agent_architecture] deploys Vector on each individual
+node for local data collection. Along with general [hardening](#7-hardening)
+requirements, the following requirements define support for this architecture:
+
+- Architecture
+  - MUST deploy as a daemon on existing nodes, one Vector process per node.
+  - MUST deploy with Vector's [default agent configuration][default_agent_configuration]
+    which largely covers the agent architecture
+    [design recommendations][agent_architecture].
+- Sizing
+  - MUST deploy as a good infrastructure citizen, giving resource priority to
+    other services on the same node.
+  - SHOULD be limited to 2 vCPUs, MUST be overridable by the user.
+  - SHOULD be limited to 2 GiB of memory per vCPU (4 GiB in this case), MUST be
+    overridable by the user.
+  - SHOULD be limited to 1 GiB of disk space, MUST be overridable by the user.
+
+### 5. Aggregator Architecture
+
+The [aggregator architecture][aggregator_architecture] deploys Vector onto
+dedicated nodes for data aggregation. Along with general [hardening](#7-hardening)
+requirements, the following requirements define support for this architecture:
+
+- Architecture
+  - SHOULD deploy as a service on dedicated nodes, Vector is the only service
+    on the node.
+  - MUST deploy with a persistent disk that is available between deployments.
+  - MUST deploy with Vector's [default aggregator configuration][default_aggregator_configuration]
+    which largely covers the aggregator architecture
+    [design recommendations][aggregator_architecture].
+    in order to achieve durability with disk buffers and source checkpoints.
+  - SHOULD deploy within one Cluster or VPC at a time.
+  - Configured Vector ports, including non-default user configured ports,
+    SHOULD be automatically accessible within the Cluster or VPC.
+  - Configured Vector sources, including non-default user configured sources,
+    SHOULD be automatically discoverable via target service discovery
+    mechanisms.
+- Sizing
+  - MUST deploy in a way that takes full advantage of all system resources.
+  - The Vector service SHOULD NOT be artificially limited with resource
+    limiters such as cgroups.
+  - SHOULD request 8 vCPUs, MUST be overridable by the user.
+  - SHOULD request 2 GiB of memory per vCPU (16 GiB in this case), MUST be
+    overridable by the user.
+  - SHOULD request 6 GiB of disk space per vCPU (48 GiB in this case), MUST be
+    overridable by the user.
+  - SHOULD use the following instances if deployed in a listed cloud.
+    - AWS SHOULD default to `c6g.2xlarge` instances with 48 GiB of EBS `io2`
+      disk space.
```

Comment:
```suggestion
    - AWS SHOULD default to `c6i.2xlarge` instances with 48 GiB of EBS `gp3`
      disk space.
```

I believe this is what we discussed?

Reply:
This was awaiting the updated going-to-prod docs but I _believe_ those are right.

Reply:
Gotcha, yeah, we can wait and see. I know we want to use `c6i` for now since `c6g` seemed to have worse performance and we don't have soak tests for it yet. I thought we discussed using `gp` disks over `io`, but I'm less sure there.

Reply:
The more I think about this as a spec - I don't think we should be specifically listing machine types, probably just CPU/RAM/DISK(speed/size)

Reply:
Yeah, I think that'd be fine too.